ui/packages/ui/src/main/webapp/components/container/backbone-container/backbone-container.tsx

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
I don't think we need to duplicate this code. We might be able to add a new package that catalog-ui-search can share with the admin ui.